docs/outlook/delegate-access.md

```diff
@@ -19,20 +19,20 @@ The following table shows supported client-server combinations for this feature,
 
```

Comment:
In line 18, we say "...Excluded combinations aren't supported." Maybe we can delete that sentence now that we've included the mobile clients?

Reply:
Good catch. Removed.